gymnasium/experimental/wrappers/__init__.py

```diff
@@ -86,3 +86,68 @@
     "VectorRecordEpisodeStatistics",
     "VectorListInfo",
 ]
+
+
+class DeprecatedWrapper(ImportError):
+    """Exception raised when an old version of a wrapper is imported."""
+
+    pass
+
+
+class InvalidVersionWrapper(ImportError):
+    """Exception raised when an invalid version of a wrapper is imported."""
+
+    pass
+
+
+def __getattr__(wrapper_name):
+    """Raise errors when importing deprecated or invalid versions of wrappers.
+
+    Args:
+        wrapper_name (str): The name of the wrapper being imported.
+
+    Raises:
+        DeprecatedWrapper: If the version is not the latest.
+        InvalidVersionWrapper: If the version doesn't exist.
+        ImportError: If the wrapper does not exist.
+    """
+    if wrapper_name == "__wrapped__" or "__test__":
+        raise AttributeError(
+            f"module 'gymnasium.experimental.wrappers' has no attribute {wrapper_name}"
+        )
+
+    base_name = wrapper_name[:-1]
+    version = wrapper_name[-1]
+
+    # Get all wrappers that start with the base wrapper name
+    wrappers = [name for name in globals().keys() if name.startswith(base_name)]
```

Comment:
I think we can use `__all__` instead of `globals`

Reply:
Sure, I hesitated with it since `__all__` is manually typed, but both works

Reply:
True, but we are only adding this for classes with a shortened import name, `gymnasiu.wrappers.NormalizeRewardV0` rather than `gymnasium.wrappers.normalize.NormalizeRewardV0`